<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JWidgetAdmin</class>
 <widget class="QWidget" name="JWidgetAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_adduser">
      <attribute name="title">
       <string>add user</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>311</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lab_loginname">
          <property name="text">
           <string>login name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lab_password">
          <property name="text">
           <string>pass word</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lab_role">
          <property name="text">
           <string>Role</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="edt_loginname"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="edt_password"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="cb_role">
          <item>
           <property name="text">
            <string>Game Player</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Game Designer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Game Server Runner</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Admin</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Root</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="lab_msg">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>311</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>200</y>
         <width>204</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_reset">
          <property name="text">
           <string>reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_add">
          <property name="text">
           <string>add</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_addgame">
      <attribute name="title">
       <string>add game</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_mainserver">
      <attribute name="title">
       <string>control main server</string>
      </attribute>
      <widget class="QPushButton" name="btn_shutdown_mainserver">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>181</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>shutdown mainserver</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_restart_mainserver">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>181</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>restart mainserver</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>140</y>
         <width>291</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
